package/MDAnalysis/core/selection.py

```diff
@@ -794,7 +794,73 @@ def _sel_with_icodes(self, vals, icodes):
         return mask
 
 
+class BoolSelection(Selection):
+    """Range selection for boolean values"""
+
+    def __init__(self, parser, tokens):
+        values = grab_not_keywords(tokens)
+        if not values:
+            values = [True]
+        val = values[0]
+        if isinstance(val, str):
+            val = val.lower()
+            if val == "false":
+                val = False
+            elif val == "true":
+                val = True
+        self.value = val
+
+    def apply(self, group):
+        vals = getattr(group, self.field)
+        return group[vals == self.value].unique
+
+
+class FloatRangeSelection(Selection):
```

Comment:
Most of this code looks like `IntRangeSelection`. Could they be unified or derived from a common base class that just changes the types?

Reply:
Yep!